routing.rst

```diff
@@ -4,38 +4,29 @@
 Routing
 =======
 
-Beautiful URLs are a must for any serious web application. This means leaving
-behind ugly URLs like ``index.php?article_id=57`` in favor of something like
-``/read/intro-to-symfony``.
-
-Having flexibility is even more important. What if you need to change the
-URL of a page from ``/blog`` to ``/news``? How many links would you need to
-hunt down and update to make the change? If you're using Symfony's router,
-the change should be trivial.
-
-.. index::
-   single: Routing; Basics
-
-.. _routing-creating-routes:
+When your application receives a request, it executes a
```

Comment:
```suggestion
When your application receives an HTTP request, it executes a
```

I find "request" a bit to vague. Routing is specifically for the HTTP protocol

Reply:
Ryan suggested the opposite 😆 

![image](https://user-images.githubusercontent.com/73419/62790187-db7be200-baca-11e9-9a59-30245721fc48.png)